src/gui/widgets/LcdWidget.cpp

```diff
@@ -294,9 +294,11 @@ void LcdWidget::initUi(const QString& name , const QString& style)
 	setWindowTitle( name );
 
 	// We should make a factory for these or something.
-	//m_lcdPixmap = new QPixmap( embed::getIconPixmap( QString( "lcd_" + style ).toUtf8().constData() ) );
-	//m_lcdPixmap = new QPixmap( embed::getIconPixmap( "lcd_19green" ) ); // TODO!!
-	m_lcdPixmap = new QPixmap( embed::getIconPixmap( QString( "lcd_" + style ).toUtf8().constData() ) );
+	//m_lcdPixmap = embed::getIconPixmap(QString("lcd_" + style).toUtf8().constData());
+	//m_lcdPixmap = embed::getIconPixmap("lcd_19green"); // TODO!!
+
+	static auto s_lcdPixmap = embed::getIconPixmap(QString("lcd_" + style).toUtf8().constData());
+	m_lcdPixmap = &s_lcdPixmap;
```

Comment:
The image this loads depends on the `style` parameter, but since it is `static`, this is initialized on the 1st call and stuck with that `style` every time afterward.

Also, the `delete m_lcdWidget;` in the destructor needs to be fixed.